tutorials/algorithms/06_grover.ipynb

```diff
@@ -0,0 +1,746 @@
+      "text/plain": [
+       "<Figure size 206.852x144.48 with 1 Axes>"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "problem.grover_operator.oracle.draw(output='mpl')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Qiskit allows for an easy construction of more complex oracles:\n",
+    "* `PhaseOracle`: for parsing logical expressions such as `'~a | b'`. This is especially useful for solving 3-SAT problems and is shown in the accompanying [Grover Examples](07_grover_examples.ipynb) tutorial.\n",
+    "\n",
+    "Here we'll use the `PhaseOracle` for the simple example of finding the state $|11\\rangle$, which corresponds to `'a & b'`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from qiskit.circuit.library.phase_oracle import PhaseOracle\n",
+    "from qiskit.exceptions import MissingOptionalLibraryError\n",
+    "\n",
+    "# `Oracle` (`PhaseOracle`) as the `oracle` argument\n",
+    "expression = '(a & b)'\n",
+    "try:\n",
+    "    oracle = PhaseOracle(expression)\n",
+    "    problem = AmplificationProblem(oracle)\n",
+    "    problem.grover_operator.oracle.draw(output='mpl')\n",
+    "except MissingOptionalLibraryError as ex:\n",
+    "    print(ex)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "You can observe, that this oracle is actually implemented with three qubits instead of two!\n",
```

Comment:
This should print the oracle, otherwise we don't see that it uses 3 qubits 😄

Reply:
Printing oracle now